queries.sql

```diff
@@ -7,4 +7,69 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon') 
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+/*update the animals table by setting the species column to unspecified*/
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] It is great that you have added the screenshot but I guess you are supposed to add all the screenshots in the pull request summary as required in the project requirements 👍 Kindly put the screenshots of this query in the pull request summary to follow the project requirements 😊 as suggested by the previous reviewer 🤔
- [ ] It would be nice as well if you replace all the new screenshots with the old screenshots in the pull request summary because i saw that you have added them but not in the pull request summary 😨